From 2356161d752c64938523b5d2388c60eeadf8273f Mon Sep 17 00:00:00 2001
From: Sam Bobrowicz <sam@elite-embedded.com>
Date: Thu, 11 Oct 2018 09:23:27 -0700
Subject: [PATCH 0466/1851] media: ov5640: fix get_light_freq on auto

commit 8ccc7217e0ab2bde87e6b6476efcfd914b31531d from
https://github.com/Xilinx/linux-xlnx.git

Light frequency was not properly returned when in auto
mode and the detected frequency was 60Hz. Fix this.

Signed-off-by: Sam Bobrowicz <sam@elite-embedded.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/i2c/ov5640.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/ov5640.c b/drivers/media/i2c/ov5640.c
index 7fb8cd22101e..ee1ce5066bd4 100644
--- a/drivers/media/i2c/ov5640.c
+++ b/drivers/media/i2c/ov5640.c
@@ -1448,6 +1448,7 @@ static int ov5640_get_light_freq(struct ov5640_dev *sensor)
 			light_freq = 50;
 		} else {
 			/* 60Hz */
+			light_freq = 60;
 		}
 	}
 
-- 
2.31.1

